### DISCUSSION TITLE
Not able to build the project in vs Code

### DISCUSSION THREAD

**Discussion by Varundeep111** [WednesdaySP/MusicPlayer#4] [Q&A] [answered]
Not able to build the project , getting errors, even when correcting  Gradle formats acc. to new guidelines and updating kotlin versions, still getting errors  related to java exceptions

Are @any able to build the project

**Comment by WednesdaySP** [comment]
wait I'll let me check.

**Comment by WednesdaySP** [comment]
@98021Varundeep  use flutter version 3.19.6, it's working

**Comment by WednesdaySP** [comment]
@98021Varundeep what's the status??

**Comment by Varundeep111** [comment]
will start working in weekend,

**Comment by Varundeep111** [comment]
@WednesdaySP 

![image](https://github.com/user-attachments/assets/f7957ea6-8896-42e0-9f76-d2ec54985bdf)
![image](https://github.com/user-attachments/assets/f5e2beb0-a7a2-4190-97ce-8d9ccaf50c1f)
![image](https://github.com/user-attachments/assets/5b5e8f8b-12a3-4f1e-a575-04947b30d157)


getting these errors, using fvm  3.19.6, but still getting errors, 

even tryied to Rebuild the entire project deleting prevoius buidls, it is working for chrome but not for mobile

**Comment by Varundeep111** [comment]
Adding the entire log file @WednesdaySP 
[errors.log](https://github.com/user-attachments/files/17799039/errors.log)

**Comment by WednesdaySP** [comment, marked as answer]
@Varundeep111  are you able to run the project or still facing the build issues??

**Comment by Varundeep111** [comment]
yes, albe to build,
